// lucide-solid/dist/source/aliases/aliases.jsx
/**
* @license lucide-solid v0.562.0 - ISC
*
* This source code is licensed under the ISC license.
* See the LICENSE file in the root directory of this source tree.
*/

// src/aliases/aliases.ts
import {
  default as default2
} from "../icons/fingerprint-pattern";
import {
  default as default3
} from "../icons/alarm-check";
import {
  default as default4
} from "../icons/alarm-minus";
import {
  default as default5
} from "../icons/alarm-plus";
import {
  default as default6
} from "../icons/arrow-down-az";
import {
  default as default7
} from "../icons/sort-desc";
import {
  default as default8
} from "../icons/arrow-down-za";
import {
  default as default9
} from "../icons/arrow-up-az";
import {
  default as default10
} from "../icons/sort-asc";
import {
  default as default11
} from "../icons/arrow-up-za";
import {
  default as default12
} from "../icons/axis-3-d";
import {
  default as default13
} from "../icons/verified";
import {
  default as default14
} from "../icons/badge-help";
import {
  default as default15
} from "../icons/between-horizonal-end";
import {
  default as default16
} from "../icons/between-horizonal-start";
import {
  default as default17
} from "../icons/book-template";
import {
  default as default18
} from "../icons/curly-braces";
import {
  default as default19
} from "../icons/subtitles";
import {
  default as default20
} from "../icons/area-chart";
import {
  default as default21
} from "../icons/bar-chart-horizontal-big";
import {
  default as default22
} from "../icons/bar-chart-horizontal";
import {
  default as default23
} from "../icons/candlestick-chart";
import {
  default as default24
} from "../icons/bar-chart-big";
import {
  default as default25
} from "../icons/bar-chart-4";
import {
  default as default26
} from "../icons/bar-chart-3";
import {
  default as default27
} from "../icons/line-chart";
import {
  default as default28
} from "../icons/bar-chart";
import {
  default as default29
} from "../icons/gantt-chart";
import {
  default as default30
} from "../icons/bar-chart-2";
import {
  default as default31
} from "../icons/pie-chart";
import {
  default as default32
} from "../icons/scatter-chart";
import {
  default as default33
} from "../icons/chrome";
import {
  default as default34
} from "../icons/alert-circle";
import {
  default as default35
} from "../icons/arrow-down-circle";
import {
  default as default36
} from "../icons/arrow-left-circle";
import {
  default as default37
} from "../icons/arrow-down-left-from-circle";
import {
  default as default38
} from "../icons/arrow-down-right-from-circle";
import {
  default as default39
} from "../icons/arrow-up-left-from-circle";
import {
  default as default40
} from "../icons/arrow-up-right-from-circle";
import {
  default as default41
} from "../icons/arrow-right-circle";
import {
  default as default42
} from "../icons/arrow-up-circle";
import {
  default as default43
} from "../icons/check-circle";
import {
  default as default44
} from "../icons/check-circle-2";
import {
  default as default45
} from "../icons/chevron-down-circle";
import {
  default as default46
} from "../icons/chevron-left-circle";
import {
  default as default47
} from "../icons/chevron-right-circle";
import {
  default as default48
} from "../icons/chevron-up-circle";
import {
  default as default49
} from "../icons/divide-circle";
import {
  default as default50
} from "../icons/gauge-circle";
import {
  default as default51
} from "../icons/minus-circle";
import {
  default as default52
} from "../icons/parking-circle-off";
import {
  default as default53
} from "../icons/parking-circle";
import {
  default as default54
} from "../icons/percent-circle";
import {
  default as default55
} from "../icons/pause-circle";
import {
  default as default56
} from "../icons/plus-circle";
import {
  default as default57
} from "../icons/play-circle";
import {
  default as default58
} from "../icons/power-circle";
import {
  default as default59
} from "../icons/help-circle";
import {
  default as default60
} from "../icons/circle-help";
import {
  default as default61
} from "../icons/circle-slashed";
import {
  default as default62
} from "../icons/stop-circle";
import {
  default as default63
} from "../icons/user-circle-2";
import {
  default as default64
} from "../icons/user-circle";
import {
  default as default65
} from "../icons/x-circle";
import {
  default as default66
} from "../icons/clipboard-signature";
import {
  default as default67
} from "../icons/clipboard-edit";
import {
  default as default68
} from "../icons/download-cloud";
import {
  default as default69
} from "../icons/upload-cloud";
import {
  default as default70
} from "../icons/code-2";
import {
  default as default71
} from "../icons/columns";
import {
  default as default72
} from "../icons/columns-settings";
import {
  default as default73
} from "../icons/table-config";
import {
  default as default74
} from "../icons/panels-left-right";
import {
  default as default75
} from "../icons/contact-2";
import {
  default as default76
} from "../icons/percent-diamond";
import {
  default as default77
} from "../icons/globe-2";
import {
  default as default78
} from "../icons/more-vertical";
import {
  default as default79
} from "../icons/more-horizontal";
import {
  default as default80
} from "../icons/file-axis-3-d";
import {
  default as default81
} from "../icons/file-badge-2";
import {
  default as default82
} from "../icons/file-json-2";
import {
  default as default83
} from "../icons/file-json";
import {
  default as default84
} from "../icons/file-bar-chart";
import {
  default as default85
} from "../icons/file-bar-chart-2";
import {
  default as default86
} from "../icons/file-line-chart";
import {
  default as default87
} from "../icons/file-pie-chart";
import {
  default as default88
} from "../icons/file-check-2";
import {
  default as default89
} from "../icons/file-code-2";
import {
  default as default90
} from "../icons/file-cog-2";
import {
  default as default91
} from "../icons/file-warning";
import {
  default as default92
} from "../icons/file-audio";
import {
  default as default93
} from "../icons/file-audio-2";
import {
  default as default94
} from "../icons/file-key-2";
import {
  default as default95
} from "../icons/file-lock-2";
import {
  default as default96
} from "../icons/file-minus-2";
import {
  default as default97
} from "../icons/file-signature";
import {
  default as default98
} from "../icons/file-edit";
import {
  default as default99
} from "../icons/file-video";
import {
  default as default100
} from "../icons/file-plus-2";
import {
  default as default101
} from "../icons/file-question";
import {
  default as default102
} from "../icons/file-search-2";
import {
  default as default103
} from "../icons/file-volume-2";
import {
  default as default104
} from "../icons/file-type-2";
import {
  default as default105
} from "../icons/file-video-2";
import {
  default as default106
} from "../icons/file-x-2";
import {
  default as default107
} from "../icons/folder-cog-2";
import {
  default as default108
} from "../icons/folder-edit";
import {
  default as default109
} from "../icons/filter-x";
import {
  default as default110
} from "../icons/filter";
import {
  default as default111
} from "../icons/git-commit";
import {
  default as default112
} from "../icons/grid-2-x-2-check";
import {
  default as default113
} from "../icons/grid-2-x-2-plus";
import {
  default as default114
} from "../icons/grid-2-x-2-x";
import {
  default as default115
} from "../icons/grid-2-x-2";
import {
  default as default116
} from "../icons/grid";
import {
  default as default117
} from "../icons/grid-3-x-3";
import {
  default as default118
} from "../icons/grab";
import {
  default as default119
} from "../icons/helping-hand";
import {
  default as default120
} from "../icons/home";
import {
  default as default121
} from "../icons/ice-cream-2";
import {
  default as default122
} from "../icons/ice-cream";
import {
  default as default123
} from "../icons/laptop-2";
import {
  default as default124
} from "../icons/layers-3";
import {
  default as default125
} from "../icons/outdent";
import {
  default as default126
} from "../icons/indent-decrease";
import {
  default as default127
} from "../icons/indent";
import {
  default as default128
} from "../icons/indent-increase";
import {
  default as default129
} from "../icons/loader-2";
import {
  default as default130
} from "../icons/unlock-keyhole";
import {
  default as default131
} from "../icons/unlock";
import {
  default as default132
} from "../icons/mail-question";
import {
  default as default133
} from "../icons/location-edit";
import {
  default as default134
} from "../icons/message-circle-question";
import {
  default as default135
} from "../icons/mic-2";
import {
  default as default136
} from "../icons/move-3-d";
import {
  default as default137
} from "../icons/alert-octagon";
import {
  default as default138
} from "../icons/pause-octagon";
import {
  default as default139
} from "../icons/x-octagon";
import {
  default as default140
} from "../icons/paintbrush-2";
import {
  default as default141
} from "../icons/panel-bottom-inactive";
import {
  default as default142
} from "../icons/sidebar-close";
import {
  default as default143
} from "../icons/panel-left-inactive";
import {
  default as default144
} from "../icons/sidebar-open";
import {
  default as default145
} from "../icons/sidebar";
import {
  default as default146
} from "../icons/panel-right-inactive";
import {
  default as default147
} from "../icons/panel-top-inactive";
import {
  default as default148
} from "../icons/layout";
import {
  default as default149
} from "../icons/edit-3";
import {
  default as default150
} from "../icons/edit-2";
import {
  default as default151
} from "../icons/plug-zap-2";
import {
  default as default152
} from "../icons/form-input";
import {
  default as default153
} from "../icons/rotate-3-d";
import {
  default as default154
} from "../icons/rows";
import {
  default as default155
} from "../icons/panels-top-bottom";
import {
  default as default156
} from "../icons/scale-3-d";
import {
  default as default157
} from "../icons/send-horizonal";
import {
  default as default158
} from "../icons/shield-question";
import {
  default as default159
} from "../icons/shield-close";
import {
  default as default160
} from "../icons/sliders";
import {
  default as default161
} from "../icons/stars";
import {
  default as default162
} from "../icons/activity-square";
import {
  default as default163
} from "../icons/arrow-down-right-square";
import {
  default as default164
} from "../icons/arrow-down-square";
import {
  default as default165
} from "../icons/arrow-down-left-square";
import {
  default as default166
} from "../icons/arrow-left-square";
import {
  default as default167
} from "../icons/arrow-down-left-from-square";
import {
  default as default168
} from "../icons/arrow-down-right-from-square";
import {
  default as default169
} from "../icons/arrow-up-left-from-square";
import {
  default as default170
} from "../icons/arrow-up-right-from-square";
import {
  default as default171
} from "../icons/arrow-up-left-square";
import {
  default as default172
} from "../icons/arrow-right-square";
import {
  default as default173
} from "../icons/arrow-up-right-square";
import {
  default as default174
} from "../icons/asterisk-square";
import {
  default as default175
} from "../icons/arrow-up-square";
import {
  default as default176
} from "../icons/scissors-square-dashed-bottom";
import {
  default as default177
} from "../icons/gantt-chart-square";
import {
  default as default178
} from "../icons/square-gantt-chart";
import {
  default as default179
} from "../icons/check-square-2";
import {
  default as default180
} from "../icons/check-square";
import {
  default as default181
} from "../icons/chevron-down-square";
import {
  default as default182
} from "../icons/chevron-left-square";
import {
  default as default183
} from "../icons/chevron-up-square";
import {
  default as default184
} from "../icons/chevron-right-square";
import {
  default as default185
} from "../icons/code-square";
import {
  default as default186
} from "../icons/kanban-square-dashed";
import {
  default as default187
} from "../icons/mouse-pointer-square-dashed";
import {
  default as default188
} from "../icons/box-select";
import {
  default as default189
} from "../icons/divide-square";
import {
  default as default190
} from "../icons/dot-square";
import {
  default as default191
} from "../icons/equal-square";
import {
  default as default192
} from "../icons/function-square";
import {
  default as default193
} from "../icons/kanban-square";
import {
  default as default194
} from "../icons/library-square";
import {
  default as default195
} from "../icons/menu-square";
import {
  default as default196
} from "../icons/m-square";
import {
  default as default197
} from "../icons/minus-square";
import {
  default as default198
} from "../icons/inspect";
import {
  default as default199
} from "../icons/parking-square-off";
import {
  default as default200
} from "../icons/parking-square";
import {
  default as default201
} from "../icons/pen-box";
import {
  default as default202
} from "../icons/edit";
import {
  default as default203
} from "../icons/pen-square";
import {
  default as default204
} from "../icons/percent-square";
import {
  default as default205
} from "../icons/pi-square";
import {
  default as default206
} from "../icons/pilcrow-square";
import {
  default as default207
} from "../icons/play-square";
import {
  default as default208
} from "../icons/plus-square";
import {
  default as default209
} from "../icons/power-square";
import {
  default as default210
} from "../icons/scissors-square";
import {
  default as default211
} from "../icons/slash-square";
import {
  default as default212
} from "../icons/sigma-square";
import {
  default as default213
} from "../icons/split-square-horizontal";
import {
  default as default214
} from "../icons/split-square-vertical";
import {
  default as default215
} from "../icons/terminal-square";
import {
  default as default216
} from "../icons/user-square-2";
import {
  default as default217
} from "../icons/user-square";
import {
  default as default218
} from "../icons/x-square";
import {
  default as default219
} from "../icons/test-tube-2";
import {
  default as default220
} from "../icons/align-center";
import {
  default as default221
} from "../icons/align-right";
import {
  default as default222
} from "../icons/align-justify";
import {
  default as default223
} from "../icons/text";
import {
  default as default224
} from "../icons/align-left";
import {
  default as default225
} from "../icons/letter-text";
import {
  default as default226
} from "../icons/text-selection";
import {
  default as default227
} from "../icons/wrap-text";
import {
  default as default228
} from "../icons/train";
import {
  default as default229
} from "../icons/palmtree";
import {
  default as default230
} from "../icons/alert-triangle";
import {
  default as default231
} from "../icons/tv-2";
import {
  default as default232
} from "../icons/school-2";
import {
  default as default233
} from "../icons/user-check-2";
import {
  default as default234
} from "../icons/user-cog-2";
import {
  default as default235
} from "../icons/user-minus-2";
import {
  default as default236
} from "../icons/user-x-2";
import {
  default as default237
} from "../icons/user-plus-2";
import {
  default as default238
} from "../icons/user-2";
import {
  default as default239
} from "../icons/users-2";
import {
  default as default240
} from "../icons/fork-knife-crossed";
import {
  default as default241
} from "../icons/fork-knife";
import {
  default as default242
} from "../icons/wallet-2";
import {
  default as default243
} from "../icons/wand-2";
export {
  default162 as ActivitySquare,
  default3 as AlarmCheck,
  default4 as AlarmMinus,
  default5 as AlarmPlus,
  default34 as AlertCircle,
  default137 as AlertOctagon,
  default230 as AlertTriangle,
  default220 as AlignCenter,
  default222 as AlignJustify,
  default224 as AlignLeft,
  default221 as AlignRight,
  default20 as AreaChart,
  default6 as ArrowDownAz,
  default35 as ArrowDownCircle,
  default37 as ArrowDownLeftFromCircle,
  default167 as ArrowDownLeftFromSquare,
  default165 as ArrowDownLeftSquare,
  default38 as ArrowDownRightFromCircle,
  default168 as ArrowDownRightFromSquare,
  default163 as ArrowDownRightSquare,
  default164 as ArrowDownSquare,
  default8 as ArrowDownZa,
  default36 as ArrowLeftCircle,
  default166 as ArrowLeftSquare,
  default41 as ArrowRightCircle,
  default172 as ArrowRightSquare,
  default9 as ArrowUpAz,
  default42 as ArrowUpCircle,
  default39 as ArrowUpLeftFromCircle,
  default169 as ArrowUpLeftFromSquare,
  default171 as ArrowUpLeftSquare,
  default40 as ArrowUpRightFromCircle,
  default170 as ArrowUpRightFromSquare,
  default173 as ArrowUpRightSquare,
  default175 as ArrowUpSquare,
  default11 as ArrowUpZa,
  default174 as AsteriskSquare,
  default12 as Axis3D,
  default14 as BadgeHelp,
  default28 as BarChart,
  default30 as BarChart2,
  default26 as BarChart3,
  default25 as BarChart4,
  default24 as BarChartBig,
  default22 as BarChartHorizontal,
  default21 as BarChartHorizontalBig,
  default15 as BetweenHorizonalEnd,
  default16 as BetweenHorizonalStart,
  default17 as BookTemplate,
  default188 as BoxSelect,
  default23 as CandlestickChart,
  default43 as CheckCircle,
  default44 as CheckCircle2,
  default180 as CheckSquare,
  default179 as CheckSquare2,
  default45 as ChevronDownCircle,
  default181 as ChevronDownSquare,
  default46 as ChevronLeftCircle,
  default182 as ChevronLeftSquare,
  default47 as ChevronRightCircle,
  default184 as ChevronRightSquare,
  default48 as ChevronUpCircle,
  default183 as ChevronUpSquare,
  default33 as Chrome,
  default60 as CircleHelp,
  default61 as CircleSlashed,
  default67 as ClipboardEdit,
  default66 as ClipboardSignature,
  default70 as Code2,
  default185 as CodeSquare,
  default71 as Columns,
  default72 as ColumnsSettings,
  default75 as Contact2,
  default18 as CurlyBraces,
  default49 as DivideCircle,
  default189 as DivideSquare,
  default190 as DotSquare,
  default68 as DownloadCloud,
  default202 as Edit,
  default150 as Edit2,
  default149 as Edit3,
  default191 as EqualSquare,
  default92 as FileAudio,
  default93 as FileAudio2,
  default80 as FileAxis3D,
  default81 as FileBadge2,
  default84 as FileBarChart,
  default85 as FileBarChart2,
  default88 as FileCheck2,
  default89 as FileCode2,
  default90 as FileCog2,
  default98 as FileEdit,
  default83 as FileJson,
  default82 as FileJson2,
  default94 as FileKey2,
  default86 as FileLineChart,
  default95 as FileLock2,
  default96 as FileMinus2,
  default87 as FilePieChart,
  default100 as FilePlus2,
  default101 as FileQuestion,
  default102 as FileSearch2,
  default97 as FileSignature,
  default104 as FileType2,
  default99 as FileVideo,
  default105 as FileVideo2,
  default103 as FileVolume2,
  default91 as FileWarning,
  default106 as FileX2,
  default110 as Filter,
  default109 as FilterX,
  default2 as Fingerprint,
  default107 as FolderCog2,
  default108 as FolderEdit,
  default241 as ForkKnife,
  default240 as ForkKnifeCrossed,
  default152 as FormInput,
  default192 as FunctionSquare,
  default29 as GanttChart,
  default177 as GanttChartSquare,
  default50 as GaugeCircle,
  default111 as GitCommit,
  default77 as Globe2,
  default118 as Grab,
  default116 as Grid,
  default115 as Grid2X2,
  default112 as Grid2X2Check,
  default113 as Grid2X2Plus,
  default114 as Grid2X2X,
  default117 as Grid3X3,
  default59 as HelpCircle,
  default119 as HelpingHand,
  default120 as Home,
  default122 as IceCream,
  default121 as IceCream2,
  default127 as Indent,
  default126 as IndentDecrease,
  default128 as IndentIncrease,
  default198 as Inspect,
  default193 as KanbanSquare,
  default186 as KanbanSquareDashed,
  default123 as Laptop2,
  default124 as Layers3,
  default148 as Layout,
  default225 as LetterText,
  default194 as LibrarySquare,
  default27 as LineChart,
  default129 as Loader2,
  default133 as LocationEdit,
  default196 as MSquare,
  default132 as MailQuestion,
  default195 as MenuSquare,
  default134 as MessageCircleQuestion,
  default135 as Mic2,
  default51 as MinusCircle,
  default197 as MinusSquare,
  default79 as MoreHorizontal,
  default78 as MoreVertical,
  default187 as MousePointerSquareDashed,
  default136 as Move3D,
  default125 as Outdent,
  default140 as Paintbrush2,
  default229 as Palmtree,
  default141 as PanelBottomInactive,
  default143 as PanelLeftInactive,
  default146 as PanelRightInactive,
  default147 as PanelTopInactive,
  default74 as PanelsLeftRight,
  default155 as PanelsTopBottom,
  default53 as ParkingCircle,
  default52 as ParkingCircleOff,
  default200 as ParkingSquare,
  default199 as ParkingSquareOff,
  default55 as PauseCircle,
  default138 as PauseOctagon,
  default201 as PenBox,
  default203 as PenSquare,
  default54 as PercentCircle,
  default76 as PercentDiamond,
  default204 as PercentSquare,
  default205 as PiSquare,
  default31 as PieChart,
  default206 as PilcrowSquare,
  default57 as PlayCircle,
  default207 as PlaySquare,
  default151 as PlugZap2,
  default56 as PlusCircle,
  default208 as PlusSquare,
  default58 as PowerCircle,
  default209 as PowerSquare,
  default153 as Rotate3D,
  default154 as Rows,
  default156 as Scale3D,
  default32 as ScatterChart,
  default232 as School2,
  default210 as ScissorsSquare,
  default176 as ScissorsSquareDashedBottom,
  default157 as SendHorizonal,
  default159 as ShieldClose,
  default158 as ShieldQuestion,
  default145 as Sidebar,
  default142 as SidebarClose,
  default144 as SidebarOpen,
  default212 as SigmaSquare,
  default211 as SlashSquare,
  default160 as Sliders,
  default10 as SortAsc,
  default7 as SortDesc,
  default213 as SplitSquareHorizontal,
  default214 as SplitSquareVertical,
  default178 as SquareGanttChart,
  default161 as Stars,
  default62 as StopCircle,
  default19 as Subtitles,
  default73 as TableConfig,
  default215 as TerminalSquare,
  default219 as TestTube2,
  default223 as Text,
  default226 as TextSelection,
  default228 as Train,
  default231 as Tv2,
  default131 as Unlock,
  default130 as UnlockKeyhole,
  default69 as UploadCloud,
  default238 as User2,
  default233 as UserCheck2,
  default64 as UserCircle,
  default63 as UserCircle2,
  default234 as UserCog2,
  default235 as UserMinus2,
  default237 as UserPlus2,
  default217 as UserSquare,
  default216 as UserSquare2,
  default236 as UserX2,
  default239 as Users2,
  default13 as Verified,
  default242 as Wallet2,
  default243 as Wand2,
  default227 as WrapText,
  default65 as XCircle,
  default139 as XOctagon,
  default218 as XSquare
};
